bin/cli.js

```diff
@@ -70,6 +70,20 @@ For more information, see https://webpack.js.org/api/cli/.`);
 		try {
 			options = require("./utils/convert-argv")(argv);
 		} catch (err) {
+			// When webpack is not installed and no args passed to the CLI
+			if (err.code === "MODULE_NOT_FOUND") {
+				let errorMessage =
+					"\n\u001b[31mwebpack not found, please install webpack using\n\t\u001b[33mnpm install --save-dev webpack\n";
+
+				if (process.env.npm_execpath !== undefined && process.env.npm_execpath.includes("yarn")) {
+					errorMessage =
+						"\n\u001b[31mwebpack not found, please install webpack using\n\t\u001b[33myarn add webpack --dev\n";
+				}
+
+				console.error(errorMessage);
+				return;
```

Comment:
Could you remove the return here and doing what we're suggesting @pranshuchittora ? After that I can do a manual test of this an verify, then we can approve and merge it

Reply:
If I remove the `return`, then the following scope will throw an error, which might freak out a user.

Here is a screenshot which shows what will happen if the func won't return.
![Screenshot from 2019-05-30 20-15-41](https://user-images.githubusercontent.com/32242596/58641030-0520cc00-8318-11e9-8e6a-014a94a3406a.png)

Reply:
IMO we must return from the func as soon as the above condition satisfies.

Reply:
So to clear things out here. Assign the process exit code as @rishabh3112 points out, shorten the stack trace by https://developer.mozilla.org/en-US/docs/Web/JavaScript/Microsoft_Extensions/Error.stackTraceLimit and make more use of coloring to make the suggestion to install webpack more visible. All good

Reply:
Done :+1: 

![Screenshot from 2019-05-30 22-26-58](https://user-images.githubusercontent.com/32242596/58649588-1a065b00-832a-11e9-8eba-d6881445cafa.png)